include/elliptics/interface.h

```diff
@@ -389,14 +389,19 @@ struct dnet_config
 
 	int			send_limit;
 
-	int			reserved_for_future_use_2[4];
+	unsigned char		reconnect_parallelism;
+
+	unsigned char		__unused_1;
+	unsigned char		__unused_2;
+	unsigned char		__unused_3;
+	int			reserved_for_future_use_2[3];
```

Comment:
Why wasn't `reserved_for_future_use_3` used/split?

Reply:
I didn't worry about such details. I'll fix.

Reply:
I called it `reserved_for_future_use_4`, because `reserved_for_future_use_3` name is already occuped.